sunpy/physics/differential_rotation.py

```diff
@@ -142,3 +141,200 @@ def solar_rotate_coordinate(coordinate, new_observer_time, new_observer_location
+    --------
+    >>> import numpy as np
+    >>> from sunpy.physics.transforms.differential_rotation import _to_norm
+    >>> out = _to_norm(np.array([-1, 0, 1]))
+    >>> out
+    array([ 0. ,  0.5,  1. ])
+    """
+    arr = np.array(arr, dtype='double')
+    arr = img_as_float(arr, force_copy=True)
+    if arr.min() < 0:
+        arr += np.abs(arr.min())
+    arr /= arr.max()
+    return arr
+
+
+def _un_norm(arr, original):
+    """
+    Helper function to un-normalise (or re-scale) an array based in
+    the values of the original array.
+
+    Parameters
+    ----------
+    arr : `~numpy.ndarray`
+        Array of floats to un-normalise with values in [0,1]
+    original : `~numpy.ndarray`
+        Original array with the min and max values
+
+    Returns
+    -------
+    arr : `~numpy.ndarray`
+        Array with values between `original.min()` and `original.max()` . Note
+        that the type of the original image is not guaranteed to be reproduced.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sunpy.physics.transforms.differential_rotation import _un_norm
+    >>> original = np.array([-1, 0, 1])
+    >>> normalised = np.array([0., 0.5, 1.])
+    >>> out = _un_norm(normalised, original)
+    >>> out
+    array([-1.,  0.,  1.])
+    """
+    level = 0 if original.min() > 0 else np.abs(original.min())
+    arr *= original.max() + level
+    arr -= level
+    return arr
+
+
+@u.quantity_input(dt=u.s)
+def _warp_sun_coordinates(xy, smap, dt):
+    """
+    Function that returns a new list of coordinates for each input coord.
+    This is an inverse function needed by the scikit-image `transform.warp`
+    function.
+
+    Parameters
+    ----------
+    xy :
+        Array from `transform.warp`
+    smap : `~sunpy.map`
+        Original map that we want to transform
+    dt : `~astropy.units.Quantity`
+        Desired interval to rotate the input map by solar differential rotation.
+
+    Returns
+    -------
+    xy2 : `~numpy.ndarray`
+        Array with the inverse transformation
+    """
+    # NOTE: The time is being subtracted - this is because this function
+    # calculates the inverse of the transformation.
+    rotated_time = smap.date - timedelta(seconds=dt.to(u.s).value)
+
+    # Calculate the hpc coords
+    x = np.arange(0, smap.dimensions.x.value)
+    y = np.arange(0, smap.dimensions.y.value)
+    xx, yy = np.meshgrid(x, y)
+    # the xy input array would have the following shape
+    # xy = np.dstack([xx.T.flat, yy.T.flat])[0]
+
+    # We start by converting the pixel to world
+    hpc_coords = smap.pixel_to_world(xx * u.pix, yy * u.pix)
+
+    # then diff-rotate the hpc coordinates to the desired time
+    rotated_coord = solar_rotate_coordinate(hpc_coords, rotated_time)
+
+    # To find the values that are behind the sun we need to convert them
+    # to HeliographicStonyhurst
+    findOccult = rotated_coord.transform_to(HeliographicStonyhurst)
+
+    with np.errstate(invalid='ignore'):
```

Comment:
If anyone knows why this is not hiding all the `RuntimeWarnings`... please, let me know it.